components/load_from_hf_hub/src/main.py

```diff
@@ -20,9 +19,9 @@ def __init__(self,
                  *_,
                  dataset_name: str,
                  column_name_mapping: dict,
-                 image_column_names: t.Optional[list],
-                 n_rows_to_load: t.Optional[int],
-                 index_column: t.Optional[str],
+                 image_column_names: list,
```

Comment:
Can we keep them as optional since they have a default or does `optional` only apply when the default is `None` by convention?

Reply:
Optional means that a `None` value can be passed in or set as a default. This is no longer the case now.

Reply:
I still think we should have optional argument (see comment above)